src/main/g8/src/main/java/com/lightbend/akka/sample/HelloAkka.java

```diff
@@ -0,0 +1,41 @@
+package com.lightbend.akka.sample;
+
+import akka.actor.ActorRef;
+import akka.actor.ActorSystem;
+import com.lightbend.akka.sample.Greeter.*;
+
+import java.io.IOException;
+
+public class HelloAkka {
+    public static void main(String[] args) {
+        final ActorSystem system = ActorSystem.create("helloakka");
+        try {
+            //#create-actors
+            final ActorRef printerActor = system.actorOf(Printer.props(), "printerActor");
+            final ActorRef howdyGreeter = system.actorOf(Greeter.props("Howdy", printerActor), "howdyGreeter");
+            final ActorRef helloGreeter = system.actorOf(Greeter.props("Hello", printerActor), "helloGreeter");
+            final ActorRef goodDayGreeter = system.actorOf(Greeter.props("Good day", printerActor), "goodDayGreeter");
+            //#create-actors
+
+            //#main-send-messages
+            howdyGreeter.tell(new WhoToGreet("Akka"), ActorRef.noSender());
+            howdyGreeter.tell(new Greet(), ActorRef.noSender());
+
+            howdyGreeter.tell(new WhoToGreet("Lightbend"), ActorRef.noSender());
+            howdyGreeter.tell(new Greet(), ActorRef.noSender());
+
+            helloGreeter.tell(new WhoToGreet("Scala"), ActorRef.noSender());
```

Comment:
I'll change it to Java everywhere so that it is not confusing for the Java users.